.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+exclude: azure-pipelines.yml
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black
+    rev: 21.4b2 # Replace by any tag/version: https://github.com/psf/black/tags
```

Comment:
können wir unten in den requirements black auch auf 21.4b2 pinnen, dass das lokale und der check online nicht auseinander laufen? zumindest in der aufgeräumten requirements.txt war es "21.5"
> black==21.5b1
>         \# via -r requirements/dev.in</span>

Reply:
sollte möglich sein.

Reply:
Black wird doch gar nicht in den requirements installiert?
Und ist es nicht eher sinnvoll die pre-commit yaml mitzuziehen? Dafür ist auch der Kommentar da oder?